ui/src/features/components/JobForm/utils.js

```diff
@@ -0,0 +1,27 @@
+import {cloneDeep} from 'lodash';
+
+export const createStateForEditJob = (job) => {
```

Comment:
the create request builder can be found in requestBuilder file.
it is a good opportunity to move the code also to this file and remove the  requestBuilder file

Reply:
moved